doc/topics/releases/oxygen.rst

```diff
@@ -273,14 +273,71 @@ The new grains added are:
 * ``iscsi_iqn``: Show the iSCSI IQN name for a host
 * ``swap_total``: Show the configured swap_total for Linux, *BSD, OS X and Solaris/SunOS
 
-Salt Minion Autodiscovery
+Salt Minion Auto-discovery
 ------------------------
 
-Salt Minion now no longer need to be configured against a specifig DNS name or IP address of a Master.
+Salt Minion now no longer need to be configured against a specific DNS name or IP address of a Master.
```

Comment:
Sorry, missed this in my last review, I think because these lines were not part of the diff before your recent commits:

```
Using auto-discovery, the Salt Minion now no longer needs to be configured
against a specific DNS name or IP address of a Master.
```